__tests__/app.test.js

```diff
@@ -646,3 +646,85 @@ describe("GET /api/users/:username", () => {
       });
   });
 });
+
+describe("PATCH /api/comments/:comment_id", () => {
+  test("status:200 returns updated comment object with incremented votes and other properties", () => {
+    return request(app)
+      .patch("/api/comments/6")
+      .send({ inc_votes: 10 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          comment_id: 6,
+          body: expect.any(String),
+          article_id: expect.any(Number),
+          author: expect.any(String),
+          created_at: expect.any(String),
+          votes: 10, // incremented from 0 to 10
+        });
+      });
+  });
+
+  test("status:200 returns updated comment object with decremented votes and other properties", () => {
+    return request(app)
+      .patch("/api/comments/1")
+      .send({ inc_votes: -10 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          comment_id: 1,
+          body: expect.any(String),
+          article_id: expect.any(Number),
+          author: expect.any(String),
+          created_at: expect.any(String),
+          votes: -10, // decremented from 16 to -10
```

Comment:
If you've sent -10 votes and started at 16, then it should be 6 votes remaining. Are you reassigning the votes to a new number or are you reassigning it to include the change?